lib/webauthn/attestation_statement/packed.rb

```diff
@@ -102,6 +113,12 @@ def matching_aaguid?(attested_credential_data_aaguid)
         end
       end
 
+      def certificate_in_use?(certificate)
+        now = Time.now
+
+        certificate.not_before < now && now < certificate.not_after
```

Comment:
This doesn't add real security since any certificate is trusted. A more thorough solution that includes this check as well would be to verify using a certificate trust story (like in Android Safetynet), but then #66 becomes required again to do this properly.

Reply:
I agree this is not thorough.
Just a step in that direction.